### PR TITLE
Feat profile functionality

### PR DESCRIPTION
**Pull request by coldpigli**

### CODE REVIEW

File: src/components/EditProfile/EditProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                <FormControl>
+											<FormLabel
+												cursor="pointer"
+												position="absolute"
+												right="0px"
+												bottom="0"
+											>
+												<Icon as={BsFillCameraFill} w='10' h='10'/>
+											</FormLabel>
+											<Input
```
Comment (nishasen) [inline src/components/EditProfile/EditProfile.jsx:106]: Fix the alignment of the code

**Review by nishasen — APPROVED** [review]
Everything looks good

File: src/components/EditProfile/EditProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+      const { firstName, lastName, portfolioUrl, bio } = values;   
+      dispatch(editUser({firstName, lastName, portfolioUrl, bio}))
+      actions.resetForm();
+    },
+  });
+
+  const uploadImage = async (image) => {
+    if (Math.round(image.size / 1024000) > 2)
+			console.log("Image cannot exceed 2mb");
+      //the below code needs to be fixed later
```
Comment (vaibhav-mahalle) [inline src/components/EditProfile/EditProfile.jsx:58]: Try and fix it before merging the pr.

**Review by vaibhav-mahalle — APPROVED** [review]
Liked the modularity and structure of your code, keep in mind to remove comments.

**Pull request merged**